projects/classification/sensor/artificial_data/clustering_settings.py

```diff
@@ -0,0 +1,22 @@
+# Param for binary data
```

Comment:
I agree. I needed to clean up the `classification` section anyway so I ended up refactoring a couple of things: 
* We have 3 sets of settings for 3 separate scripts / phases: `settings/htm_network.py` contains params for running the network with `run_htm_network.py`, regardless of wether we are using accelerometer data or artificial data. `settings/acc_data.py` contains settings for generating the cleaned up accelerometer data files and has clustering params tuned for the accelerometer data. Finally, `settings/artificial_data.py` contains settings for generating the artificial data files and has clustering params tuned for the artificial data.
* Updated the README
* Removed duplicated accelerometer data files